app_flutter/lib/build_dashboard_page.dart

```diff
@@ -156,7 +240,13 @@ class BuildDashboardPageState extends State<BuildDashboardPage> {
           title: statusTable[_buildState.isTreeBuilding],
           backgroundColor: colorTable[_buildState.isTreeBuilding],
           actions: <Widget>[
+            PopupMenuButton<String>(
+              tooltip: 'Task Status Key',
+              child: const Icon(Icons.vpn_key),
```

Comment:
My first impression of the key was that it was a menu to store/interact with secrets. Maybe `info` or some book icon would be more clear?